package.json

```diff
@@ -13,7 +13,7 @@
 	},
 	"homepage": "https://github.com/Autodesk-AutoCAD/AutoLispExt",
 	"engines": {
-		"vscode": "^1.31.1"
+		"vscode": "^1.50.0"
```

Comment:
This will make the extension only support 1.50.0 or the higher version VSCODE

Reply:
It's because of the following error:
![image](https://user-images.githubusercontent.com/57521938/112923789-cbf84500-90c3-11eb-90ea-9c51071bda06.png)

Shall we apply my change or lower the version of types/vscode?